### ISSUE TITLE
Error compiling amazonka

### ISSUE THREAD

**Issue by newhoggy** [brendanhay/amazonka#446]
```
$ ghc --version
The Glorious Glasgow Haskell Compilation System, version 8.2.2
$ cabal --version
cabal-install version 2.0.0.1
compiled using version 2.0.1.0 of the Cabal library
$ git clone git@github.com:haskell-works/kuneiform.git
$ cd kuneiform
$ cabal sandbox init
$ cabal update
$ cabal install --jobs=8 --max-backjumps=100000 --only-dependencies
...
cabal: Entering directory '/var/folders/8d/3xbnllbx76gcbk3wwy086vlm0000gn/T/cabal-tmp-20383/amazonka-1.5.0'
Configuring amazonka-1.5.0...
Preprocessing library for amazonka-1.5.0..
Building library for amazonka-1.5.0..
[ 1 of 10] Compiling Network.AWS.Data ( src/Network/AWS/Data.hs, dist/dist-sandbox-1ea8757a/build/Network/AWS/Data.o )
[ 2 of 10] Compiling Network.AWS.EC2.Metadata ( src/Network/AWS/EC2/Metadata.hs, dist/dist-sandbox-1ea8757a/build/Network/AWS/EC2/Metadata.o )
[ 3 of 10] Compiling Network.AWS.Auth ( src/Network/AWS/Auth.hs, dist/dist-sandbox-1ea8757a/build/Network/AWS/Auth.o )

src/Network/AWS/Auth.hs:395:35: warning: [-Wname-shadowing]
    This binding for ‘region’ shadows the existing binding
      imported from ‘Network.AWS.EC2.Metadata’ at src/Network/AWS/Auth.hs:79:1-31
      (and originally defined at src/Network/AWS/EC2/Metadata.hs:377:1-6)
    |
395 | fromEnvKeys access secret session region =
    |                                   ^^^^^^
[ 4 of 10] Compiling Network.AWS.Internal.Body ( src/Network/AWS/Internal/Body.hs, dist/dist-sandbox-1ea8757a/build/Network/AWS/Internal/Body.o )

src/Network/AWS/Internal/Body.hs:14:1: warning: [-Wunused-imports]
    The import of ‘Control.Applicative’ is redundant
      except perhaps to import instances from ‘Control.Applicative’
    To import instances alone, use: import Control.Applicative()
   |
14 | import           Control.Applicative
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[ 5 of 10] Compiling Network.AWS.Internal.Logger ( src/Network/AWS/Internal/Logger.hs, dist/dist-sandbox-1ea8757a/build/Network/AWS/Internal/Logger.o )
[ 6 of 10] Compiling Network.AWS.Env  ( src/Network/AWS/Env.hs, dist/dist-sandbox-1ea8757a/build/Network/AWS/Env.o )
src/Network/AWS/Env.hs:195:24: error:
    Variable not in scope: conduitManagerSettings :: ManagerSettings
    |
195 |     liftIO (newManager conduitManagerSettings)
    |                        ^^^^^^^^^^^^^^^^^^^^^^
cabal: Leaving directory '/var/folders/8d/3xbnllbx76gcbk3wwy086vlm0000gn/T/cabal-tmp-20383/amazonka-1.5.0'
```

**Comment by phadej** [comment]
I noted that too. I'll check in a second.

**Comment by phadej** [comment]
Most likely: `http-conduit-2.3.0` https://hackage.haskell.org/package/http-conduit-2.3.0/changelog (`amazonka` has optimistic `http-conduit < 3` now)

**Comment by phadej** [comment]
## unconstrained

![screenshot from 2018-01-30 13-42-06](https://user-images.githubusercontent.com/51087/35564846-410e960c-05c4-11e8-9025-be582c182428.png)

## http-conduit >=2.3

![screenshot from 2018-01-30 13-47-47](https://user-images.githubusercontent.com/51087/35564836-38e0fe7a-05c4-11e8-8f12-a3939cb9f1e3.png)

## http-conduit <2.3

*pending*

---

I'll make revisions for 5 latest versions if the `<2.3` seems to fix them.

**Comment by brendanhay** [comment]
Fixed in #441
